src/main/java/net/snowflake/client/core/json/BytesConverter.java

```diff
@@ -46,6 +46,9 @@ public byte[] getBytes(Object obj, int columnType, int columnSubType, Integer sc
               .toByteArray();
         case Types.VARCHAR:
         case Types.CHAR:
+        case Types.STRUCT:
+        case Types.ARRAY:
+        case 50003: // VECTOR
```

Comment:
Don't we have a const for it in `SnowflakeUtil`?

Reply:
I wasn't aware of that - thanks, used const